HW_5/task_3.js

```diff
@@ -0,0 +1,50 @@
+/*
+5*. Напишите функцию findMissingNumber(arr), которая принимает массив чисел от 1 до N (исключая одно число) 
+  и возвращает пропущенное число. Массив не отсортирован и может содержать дубликаты. 
+  Решите эту задачу, используя эффективные методы массива. */
+
+
+
+function findMissingNum(arr) {    
+    arr = [...new Set(arr)];        
+    let sum = arr.reduce((sum, num) => sum + num, 0);    
+    let goodSum = 0;    
+    for (let i = 1; i <= arr[arr.length - 1]; i++) {

```

Comment:
`Решите эту задачу, используя эффективные методы массива.` :(
```suggestion
    for (let i = 1; i <= arr[arr.length - 1]; i++) {
```